roman.py

```diff
@@ -33,21 +33,21 @@ def get_roman_digit(number, base):
     if quotient >= 5:
         # do something including the half base
         if quotient == 9:
-            sys.stdout.write('%s%s' % (roman_prefix[(base*10)], roman_digit[(base*10)])) 
+            sys.stdout.write('%s%s' % (ROMAN_PREFIX[(base*10)], ROMAN_DIGIT[(base*10)])) 
```

Comment:
you'd probably be better off using `print`

explicit `sys.stdout.write` is really only a good move when you are simultaneously doing file IO (and that's where print is set to go) and still want to send stuff to stdout.

these details may help: http://stackoverflow.com/questions/3263672/python-the-difference-between-sys-stdout-write-and-print